scripts/python/generate_dbt_yaml.py

```diff
@@ -292,6 +292,33 @@ def omop_docs_to_dbt_config(
             }
             tests.append(specific_test)
 
+    # Add dbt_expectations tests
+    tests.append("dbt_expectations.expect_column_to_exist")
+    if doc_container.datatype.lower() == "integer":
+        tests.append({
+            "dbt_expectations.expect_column_values_to_be_in_type_list": {
+                "column_type_list": "{{ get_type_variants(['bigint', 'integer']) }}"
+            }
+        })
+    elif doc_container.datatype.lower() in {"date", "float"}:
+        tests.append({
+            "dbt_expectations.expect_column_values_to_be_in_type_list": {
+                "column_type_list": f"{{{{ get_equivalent_types('{doc_container.datatype.lower()}') }}}}"
+            }
+        })
+    elif doc_container.datatype.lower() == "datetime":
+        tests.append({
+            "dbt_expectations.expect_column_values_to_be_in_type_list": {
+                "column_type_list": "{{ get_equivalent_types('timestamp') }}"
+            }
+        })
+    elif doc_container.datatype.lower().startswith("varchar"):
```

Comment:
for now i am ignoring the varchar lengths specified in the CDM spec.  i need to decide how to enforce these in the dbt models, and then add back tests appropriately.